src/test/java/uk/gov/pay/ledger/transaction/dao/TransactionDaoIT.java

```diff
@@ -35,6 +35,7 @@ public void shouldInsertTransaction() {
                 .withExternalMetadata(ImmutableMap.of("key1", "value1", "anotherKey", ImmutableMap.of("nestedKey", "value")))
                 .withTransactionType("PAYMENT")
                 .withLive(true)
+                .withGatewayTransactionId("gateway_transaction_id")
```

Comment:
[minor] Shouldn't we have an assertion that verifies this field too? (I can't see it there at the moment)

Reply:
It is going to be included. PR wasn't ready for review.

Reply:
No worries, as I said it's all my fault